### PR TITLE
add unofficial COPR repostitory to README

### PR DESCRIPTION
**Pull request by uriesk**
I built the package on copr from both the 1.0.7 tag and current master branch.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -42,6 +42,13 @@ brief instructions for your distribution included here.
   [tagged releases](https://aur.archlinux.org/packages/dracut-crypt-ssh/) and the
   [git HEAD](https://aur.archlinux.org/packages/dracut-crypt-ssh-git/).
 
+- An inofficial [COPR repository](https://copr.fedorainfracloud.org/coprs/uriesk/dracut-crypt-ssh/)
```
Comment (ahesford) [inline README.md]:
```suggestion
- An unofficial [COPR repository](https://copr.fedorainfracloud.org/coprs/uriesk/dracut-crypt-ssh/)
```

**Review by ahesford — CHANGES_REQUESTED** [review]
Thanks for picking this up. Please fix the typo and we can merge.

File: README.md (hunk 1 above)
Comment (uriesk) [reply README.md]: done

**Comment by ahesford** [comment]
Thanks again!

**Pull request closed without merging**